### PR TITLE
feat: add MyC artworks to the my collection tab on collector profile

### PR DESCRIPTION
**Pull request by dariakoko**
The type of this PR is: **Feat**

This PR solves [CX-3246]

### Description

I reused MyCollectionRouteRefetchContainer, changed only the name of the container, component, query and other things like that 
**_I didn't hide the changes behind a FF_**, I think if it's off, the user won't be able to reach this page anyway. I would test the experience with FF off during the QA session when all work is done.

| Web | mWeb |
| --- | --- |
| image | image |

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CX-3246]: https://artsyproduct.atlassian.net/browse/CX-3246?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by MrSltun — APPROVED** [review]
🌟

**Review by MounirDhahri — APPROVED** [review]
LGTM!
I would have prefered though if you have imported my collection from the folder where it exists right now. this way, in case we make changes there, they will be in both places. also, we won't loose any git history

**Comment by dariakoko** [comment]
@MounirDhahri agree! will do it now
Yesterday had some issues with importing the module, but I get that the issue was now

File: src/Apps/Settings/Routes/MyCollection/MyCollectionRoute.tsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ import {
 import { MyCollectionRoute_me$data } from "__generated__/MyCollectionRoute_me.graphql"
 import { EmptyMyCollectionPage } from "./Components/EmptyMyCollectionPage"
 
-interface MyCollectionRouteProps {
+export interface MyCollectionRouteProps {
```
Comment (dariakoko) [inline src/Apps/Settings/Routes/MyCollection/MyCollectionRoute.tsx:32]: We need this export to be able to reuse the component in another file

File: src/Apps/CollectorProfile/Routes/MyCollection/CollectorProfileMyCollectionRoute.tsx [hunk 2]
```diff
@@ -1,14 +1,3 @@
...
-
-  if (!isCollectorProfileEnabled) {
-    return null
-  }
-
-  return <Text>Ja Hallo!</Text>
-}
-
-export const CollectorProfileMyCollectionRouteFragmentContainer = CollectorProfileMyCollectionRoute
+export const CollectorProfileMyCollectionRouteRefetchContainer = MyCollectionRouteRefetchContainer
```
Comment (olerichter00) [inline src/Apps/CollectorProfile/Routes/MyCollection/CollectorProfileMyCollectionRoute.tsx:3]: **question:** Can we just remove this file and import `MyCollectionRouteRefetchContainer` directly wherever it is needed?

Comment (dariakoko) [reply src/Apps/CollectorProfile/Routes/MyCollection/CollectorProfileMyCollectionRoute.tsx:3]: I'd keep this file just to make it clear that this is a different thing
When cleaning up the FF I'd remove this file and just move the MyCollectionRouteRefetchContainer here + renamed some variables, queries, etc

Comment (olerichter00) [reply src/Apps/CollectorProfile/Routes/MyCollection/CollectorProfileMyCollectionRoute.tsx:3]: Sounds good 👍  To be honest, I cannot tell what `CollectorProfileMyCollectionRoute` is supposed to be 🤔.

Comment (dariakoko) [reply src/Apps/CollectorProfile/Routes/MyCollection/CollectorProfileMyCollectionRoute.tsx:3]: MyCollection, but in the collector profile 😅

Comment (dariakoko) [reply src/Apps/CollectorProfile/Routes/MyCollection/CollectorProfileMyCollectionRoute.tsx:3]: collector-profile/my-collection

Comment (olerichter00) [reply src/Apps/CollectorProfile/Routes/MyCollection/CollectorProfileMyCollectionRoute.tsx:3]: Ooooh, I see :)

**Pull request merged**